build.sbt

```diff
@@ -309,18 +309,28 @@ lazy val dagShared = (project in file("modules/dag-shared"))
   )
 
 lazy val rosetta = (project in file("modules/rosetta"))
-  .dependsOn(shared % "compile->compile;test->test")
+  .enablePlugins(AshScriptPlugin)
+  .enablePlugins(JavaAppPackaging)
```

Comment:
Does this module actually use these plugins?

Reply:
I tested something and left it. It's actually unused now